tests/storage/outofprocess_test.py

```diff
@@ -31,6 +31,7 @@
 import weakref
 
 from contextlib import contextmanager
+from pathlib import Path
```

Comment:
We don't use pathlib in vdsm for 2 reasons:
- It did not exist in python 2.7, and and vdsm was ported to python 3 only in 2020
- I don't see any advantage in using it, and syntax like:

      path / "file"

  for joining "file" to a path is actually worse. "/" should be used only for 
  division.

Reply:
Well, the '/' syntax is optional, it could as well be parameters of the Path constructor: `Path(path, "file)`.

In general as I wrote before, for joining there is not much difference between `os.path` and `pathlib`, but there are cases where working with `os.path` relies on nested calls and string manipulation, whereas pathlib deals with it much more elegantly and readably.

I know why it wasn't used before, but introducing a new library when we do new code or when code is revisited is nice, overtime we will see libraries removed/modernized without having to do a massive refactor.

Anyway, I will not push for the change, I'll change it back to `os.path` for now. We can re-discuss this in the future if the need rises.

Reply:
Introducing new library in a big code base creates yet another way to do the same thing. This makes the code harder to work with. It can make sense if the new way is much better and solve actual issues.

Good example is using fstring in error messages - most of the code uses % which is error prone. In newer code we moved to str.format() which is less fragile. Since we moved to python 3, we use fstrings in error messages, which much less error prone, more clear and less verbose.